packages/js-sdk/src/sandbox/signature.ts

```diff
@@ -12,36 +10,62 @@ import crypto from 'node:crypto'
  * @param expirationInSeconds Optional signature expiration time in seconds.
  */
 
-
 interface SignatureOpts {
-    path: string
-    operation: 'read' | 'write'
-    user: string
-    expirationInSeconds?: number
-    envdAccessToken?: string
+  path: string
+  operation: 'read' | 'write'
+  user: string
+  expirationInSeconds?: number
+  envdAccessToken?: string
+}
+
+// Cross-platform crypto implementation
+async function sha256(data: string): Promise<string> {
+  // Use Node.js crypto if WebCrypto is not available
```

Comment:
nit: do you think we can make the comment proper? We are using Node Crypto as the default option

Reply:
I moved the logic a bit other ways (use WebCrypto if available, node:crypto by default)